packages/monitoring-system/data-processor/test/unit/data-processors/cloud-logs-data-processor.spec.ts

```diff
@@ -326,8 +479,8 @@ describe('Cloud Logs Processor', () => {
               execution_id: 'g36ouppwsu6z',
               action_type: 'ISSUE_ADD_LABELS',
               timestamp: 1596118668017,
-              destination_object: 'ISSUE_python-ndb_googleapis_org_489',
-              destination_repo: 'python-ndb_googleapis_org',
+              destination_object: 'ISSUE_python-ndb_googleapis_489',
```

Comment:
The primary key for `GitHub_Repository` docs was changed to not include the `owner_type` (i.e. `user` or `org`) since:

1. It doesn't make a difference as all usernames/orgnames are unique
2. We don't always have that information in repo-automation-bots

The document will still include this information wherever available, but we simply won't have it in the primary key